lib/ansible/modules/network/aci/msc_site.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: msc_site
+short_description: Manage sites
+description:
+- Manage sites on Cisco ACI Multi-Site.
+author:
+- Dag Wieers (@dagwieers)
+version_added: '2.8'
+options:
+  apic_password:
+    description:
+    - The password for the APICs.
+  apic_site_id:
+    description:
+    - The site ID of the APICs.
+  apic_username:
+    description:
+    - The username for the APICs.
+    default: admin
+  site_id:
+    description:
+    - The ID of the site.
+    required: yes
+  site:
+    description:
+    - The name of the site.
+    required: yes
+    aliases: [ name, site_name ]
+  labels:
```

Comment:
Does this auto-create labels if they aren't created?

Reply:
Not yet, but that is definitely the intention.